pallets/eth-bridge/src/benchmarking.rs

```diff
@@ -164,6 +168,66 @@ fn setup_active_tx<T: Config>(
     });
 }
 
+fn setup_new_active_tx<T: Config>(
+    tx_id: EthereumId,
+    num_confirmations: u32,
+    sender: Validator<<T as pallet_avn::Config>::AuthorityId, T::AccountId>,
+) {
+    setup_active_tx::<T>(
+        tx_id,
+        num_confirmations,
+        sender,
+        BoundedVec::default(),
+        BoundedVec::default(),
+    );
+}
+fn setup_active_tx_with_failure_corroborations<T: Config>(
+    tx_id: EthereumId,
+    num_confirmations: u32,
+    sender: Author<T>,
+    authors: Vec<Author<T>>,
+    author: &Author<T>,
+) {
+    let mut local_authors: Vec<Author<T>> = authors.to_vec();
+    local_authors.retain(|author_from_vec| author_from_vec.account_id != sender.account_id);
+    local_authors.retain(|author_from_vec| author_from_vec.account_id != author.account_id);
+
+    let quorum = authors.len() - authors.len() * 2 / 3;
+
+    let ( num_failure_corroborations, num_successful_corroborations ) = get_num_corroborations::<T>(quorum);
+
+    println!("{:?},{:?},{:?}", local_authors, author, sender);
```

Comment:
we should remove this, otherwise we will get compilation issues when adding the benchmark feature